backend/danswer/semantic_search/semantic_search.py

```diff
@@ -66,6 +66,11 @@ def get_default_reranking_model() -> CrossEncoder:
     return _RERANK_MODEL
 
 
+def warm_up_models() -> None:
+    get_default_embedding_model().encode("Danswer is so cool")
+    get_default_reranking_model().predict(("What is Danswer", "Enterprise QA"))  # type: ignore
```

Comment:
Why do we need a # type: ignore?

Reply:
And do we need to actually call encode / predict, or can we just call `get_default_embedding_model()` / `get_default_reranking_model()` to load the models in

Reply:
gotta actually call the predict, the models are lazy loaded on inference :P

Reply:
tried to look for an arg to eager load them but didn't see it.

Reply:
We need a #type ignore because they overwrote predict to do something else that doesn't follow the signature I guess... its in the model library, not our issue